core/src/epicli/cli/engine/UpgradeEngine.py

```diff
@@ -40,13 +40,9 @@ def backup_build(self):
 
         # no backup dir so use the latest
         backup_dir_name = f'backup_{int(round(time.time() * 1000))}'
-        backup_temp_dir = os.path.join(self.build_dir, '../', backup_dir_name)
         self.backup_build_dir = os.path.join(self.build_dir, backup_dir_name )
-        self.logger.info(f'Backing up old build files to "{self.backup_build_dir}"')
-        shutil.move(self.build_dir, backup_temp_dir, copy_function = shutil.copytree) 
-        if not os.path.exists(self.build_dir):
-            os.mkdir(self.build_dir)        
-        shutil.move(backup_temp_dir, self.backup_build_dir, copy_function = shutil.copytree)
+        self.logger.info(f'Backing up build dir to "{self.backup_build_dir}"')
+        shutil.copytree(self.build_dir, self.backup_build_dir)
```

Comment:
Please take a look at this part of Python 3.7 documentation (https://docs.python.org/3.7/library/shutil.html#shutil.copytree):
```
shutil.copytree(src, dst, symlinks=False, ignore=None, copy_function=copy2, ignore_dangling_symlinks=False)
Recursively copy an entire directory tree rooted at src, returning the destination directory. The destination directory, named by dst, must not already exist;
```
It clearly states that the `directory must not exist`. Of course the name is a timestamp basically, but personally I would be extra paranoid here (but that just maybe me idk, please ignore if you think it's not a valid suggestion) \\:D/.